chirp/drivers/icr6.py

```diff
@@ -0,0 +1,794 @@
+"""Icom IC-R6 Driver"""
+# Copyright 2024 John Bradshaw Mi0SYN <john@johnbradshaw.org>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+# TODO:
+# Channel: Frequency setting
+# Channel: TSQL freq or DTCS code with polarity
+# Channel: Skip scan settings
+# Channel: Comment (not sure if supported in ICF)
```

Comment:
This is handled internally by CHIRP (in the metadata). Unless the radio can store it (I doubt it) you should just let CHIRP's base infrastructure do it for you.

Reply:
Not visible in CS-R6 either so have removed all those comments.